styleguide/source/_patterns/02-molecules/article-metadata.twig

```diff
@@ -1,4 +1,2 @@
-<div class="ama__metadata ama__topic_article-preview_metadata">
-   <span>{{ date|default('Jul 28, 2017') }}</span> |
-   <span class="topic_article-preview_metadata_read-time">{{ readtime|default("5 min read") }}</span>
-</div>
+<span>{{ date|default('Jul 28, 2017') }}</span> |
```

Comment:
Would there ever be a case where the date would appear but the time would not time and therefore we would not want the `|` to appear? Do we need an `if` statement here to account for that?

Reply:
This follows the SG1 pattern. I don't know if time would appear without a date